packages/frontend/src/components/ProjectConnection/WarehouseForms/defaultValues.ts

```diff
@@ -0,0 +1,130 @@
+import { WarehouseTypes } from '@lightdash/common';
+
+import type {
+    CreateBigqueryCredentials,
+    CreateDatabricksCredentials,
+    CreatePostgresCredentials,
+    CreateRedshiftCredentials,
+    CreateSnowflakeCredentials,
+    CreateTrinoCredentials,
+} from '@lightdash/common';
+
+// https://github.com/lightdash/lightdash/blob/427088ec09485f334f463b6026025d4287ad302b/packages/backend/src/models/ProjectModel/ProjectModel.ts#L156
+// only undefined and empty string are considered empty values
+
+export const BigQueryDefaultValues: Partial<CreateBigqueryCredentials> = {
+    type: WarehouseTypes.BIGQUERY,
+    dataset: '',
+    project: '',
+    location: '',
+    // @ts-expect-error we need to set it as empty string to avoid overwritting saved value
+    keyfileContents: '',
+    executionProject: '',
+    timeoutSeconds: 300,
+    priority: 'interactive',
+    retries: 3,
+    maximumBytesBilled: 1000000000,
+    // we need to set it as undefined to avoid overwritting saved value
```

Comment:
can you explain these two comments further?

Reply:
is it because of the backend, that you commented above?
```
                const isSecretMissingInNewConfig =
                    newConfigSecretValue === undefined ||
                    newConfigSecretValue === ''; // Null values are not considered missing
```

Reply:
yes

type of `startOfWeek` is number which is why i set it to `undefined` instead of empty string 🫠